File: coding_round2_struct/Overlapping_Intervals/starter_code/solution.java
import java.util.*;
public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        if(!sc.hasNextInt()) return;
        int n = sc.nextInt();
        long[][] iv = new long[n][2];
        for(int i=0;i<n;i++){ iv[i][0]=sc.nextLong(); iv[i][1]=sc.nextLong(); }
        // TODO: merge intervals and print lines
    }
}

File: coding_round2_struct/Overlapping_Intervals/starter_code/solution.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int n; if(scanf("%d", &n)!=1) return 0;
    long long* l = malloc(sizeof(long long)*n);
    long long* r = malloc(sizeof(long long)*n);
    for(int i=0;i<n;i++) scanf("%lld %lld", &l[i], &r[i]);
    // TODO: merge intervals and print lines
    free(l); free(r);
    return 0;
}
